app/models/licence-version.model.js

```diff
@@ -23,6 +23,18 @@ class LicenceVersionModel extends BaseModel {
           from: 'licenceVersions.licenceId',
           to: 'licences.id'
         }
+      },
+      purposes: {
+        relation: Model.ManyToManyRelation,
+        modelClass: 'purpose.model',
+        join: {
+          from: 'licenceVersions.id',
+          through: {
+            from: 'licenceVersionPurposes.licenceVersionId',
+            to: 'licenceVersionPurposes.purposeId'
+          },
+          to: 'purposes.id'
+        }
```

Comment:
As a way of ensuring we don't break this relationship, it would be ideal if we also had a test in `test/models/licence-version.model.test.js` that covered this relationship.